pkgs/development/libraries/sqlite/default.nix

```diff
@@ -1,4 +1,4 @@
-{ lib, stdenv, fetchurl, zlib, readline, ncurses
+{ lib, stdenv, fetchurl, zlib, readline, ncurses, fetchpatch
```

Comment:
Leftover `fetchpatch`? The change does not introduce new calls.

Reply:
Damnit yes well spotted